src/app/index/metadata.rs

```diff
@@ -147,6 +202,11 @@ fn read_ape(path: &Path) -> Result<SongTags> {
 	let year = tag.item("Year").and_then(read_ape_i32);
 	let disc_number = tag.item("Disc").and_then(read_ape_x_of_y);
 	let track_number = tag.item("Track").and_then(read_ape_x_of_y);
+	let lyricist = tag.item("LYRICIST").and_then(read_ape_string);
```

Comment:
Assuming the names are not case sensitive, can you keep the case consistent between them?

Reply:
Surprisingly they are. (I tried kid3 and mp3tag.) Let me know.